Doc/using/cmdline.rst

```diff
@@ -119,6 +124,7 @@ source.
    .. versionchanged:: 3.4
       namespace packages are also supported
 
+.. _cmdarg-dash:
```

Comment:
Is it possible the `:` at the end of the name is not necessary and that is what's affecting the font?

Reply:
Can you be little more specific on affecting the font part? On removing `:` at the end I get `/home/cpython/Doc/using/cmdline.rst:127: WARNING: malformed hyperlink target.` . To be honest my knowledge on Sphinx markdown is low. I took this syntax as an exact copy from https://bugs.python.org/issue33095#msg319812 . I was trying to make them hyperlinks so that they can be referenced I might be wrong on the syntax here.

Thanks

Reply:
Initially I was trying out `.. cmdoption:: -` and `.. cmdoption:: script` removing `describe::` which works fine but there was an explicit note on the bpo message to use `.. _cmdarg-dash:` so that they are hyperlinkable like option ones without looking the same.

> For making "-" and "<script>" hyperlinkable, you'll need to add a label definition to the Sphinx source code immediately before those description sections. I'd suggest ".. _cmdarg-dash:" and ".. _cmdarg-script:" so the resulting anchors look similar to the option ones, without being the same.